### DISCUSSION TITLE
How to see and compare the old feed

### DISCUSSION THREAD

**Discussion by falsandtru** [community/community#66207] [Other Feature Feedback, Questions, & Ideas]
We can see and compare the old feed on https://github.com/dashboard-feed.

Note that this old feed will be deleted. The only effectual way is to revert the official feed.

**Comment by beeman** [comment]
The old feed is there! 🥺

If I find some time I'll create a Chrome extension that swaps this in the UI - of someone doesn't beat me to it 🙌

**Reply by falsandtru** [reply]
It is useless. This old feed will be deleted. The only effectual way is to revert the official feed.
